packages/member-management/Classes/Controller/BackendMemberController.php

```diff
@@ -7,22 +7,41 @@
 use Psr\Http\Message\ResponseInterface;
 use TYPO3\CMS\Backend\Attribute\AsController;
 use TYPO3\CMS\Backend\Template\ModuleTemplateFactory;
+use TYPO3\CMS\Core\Pagination\ArrayPaginator;
+use TYPO3\CMS\Core\Pagination\SlidingWindowPagination;
 use TYPO3\CMS\Extbase\Mvc\Controller\ActionController;
+use TYPO3Incubator\MemberManagement\Domain\Repository\MemberRepository;
 
 #[AsController]
 final class BackendMemberController extends ActionController
 {
     public function __construct(
         protected readonly ModuleTemplateFactory $moduleTemplateFactory,
+        private readonly MemberRepository $memberRepository,
     ) {
     }
 
     public function indexAction(): ResponseInterface
     {
         $moduleTemplate = $this->moduleTemplateFactory->create($this->request);
+        $members = $this->memberRepository->findAll()->toArray();
+        $itemsPerPage = 20;
+        $currentPage = $this->request->hasArgument('currentPageNumber')
+            ? (int)$this->request->getArgument('currentPageNumber')
+            : 1;
+        $maximumLinks = 15;
+        $paginator = new ArrayPaginator($members, $currentPage, $itemsPerPage);
```

Comment:
I think we can omit the `toArray` call and use the `QueryResultPaginator` instead. WDYT?